editor/editor_interface.cpp

```diff
@@ -336,6 +337,22 @@ void EditorInterface::popup_property_selector(Object *p_object, const Callable &
 	property_selector->connect(SNAME("canceled"), canceled_callback, CONNECT_DEFERRED);
 }
 
+void EditorInterface::popup_quick_open(const Callable &p_callback, const TypedArray<StringName> &p_base_types) {
+	StringName required_type = SNAME("Resource");
+	Vector<StringName> base_types;
+	if (p_base_types.is_empty()) {
+		base_types.append(required_type);
+	}
+	for (const StringName &type : p_base_types) {
```

Comment:
You could add `else` to the above condition; no need to run the loop if the  array was empty.